compiler/rustc_mir/src/transform/check_consts/validation.rs

```diff
@@ -789,10 +789,10 @@ impl Visitor<'tcx> for Validator<'mir, 'tcx> {
         }
     }
 
+    #[instrument(skip(self))]
```

Comment:
What's this about?

Reply:
`Validator` doesn't derive `Debug`, which I presume is necessary to not skip it.

Reply:
Eh... I have never seen `instrument` attributes before so this answer is way over my head. ;)

Reply:
I suppose this is https://docs.rs/tracing-attributes/0.1.11/tracing_attributes/attr.instrument.html

Reply:
yea, this is just convenience code for constructing `tracing::Span` and dropping them when the function finishes running